CHANGES.txt

```diff
@@ -27,6 +27,9 @@ Changes
 Fixes
 =====
 
+ - Improved error message when trying to update an element of an array using 
+   the index in the array.
```

Comment:
would skip the ".. using.." part, afaik there is no other way updating an array element